lesson-14/src/index.js

```diff
@@ -0,0 +1,34 @@
+const products = document.querySelectorAll('li')
+const productsArray = [...products]
+
+productsArray[1].style.color = 'yellow'
+productsArray.forEach((element) => {
+  element.style.textDecoration = 'underline'
+})
+
+const article = document.querySelector('aside')
+
+function addButton(textInButton, eventForButton) {
+  const newButton = document.createElement('button')
+  newButton.innerText = textInButton
+  newButton.onclick = eventForButton
+  article.appendChild(newButton)
+}
+
+function changeFirstProduct() {
+  productsArray[0].innerText = 'Смартфон'
+}
+
+function hiddenThirdProduct() {
+  productsArray[2].classList.toggle('is-hidden')
```

Comment:
```suggestion
  productsArray[2].classList.add('is-hidden')
```

по заданию просто скрыть, `toggle` при повторном нажатии покажет - текст на кнопке ложный получается

вот если бы он поменялся на "Показать третий продукт" - было бы совсем другое дело :)